driver/src/main/com/mongodb/client/gridfs/GridFSBuckets.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2015 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client.gridfs;
+
+
+import com.mongodb.client.MongoDatabase;
+
+/**
+ * A factory for GridFSBucket instances.
+ *
+ * @since 3.1
+ */
+public final class GridFSBuckets {
```

Comment:
Yes, this is how a user can create the bucket and follows the MongoClients factory class in the async driver.

Done.